docs/uninstall/conda.rst

```diff
@@ -11,25 +11,13 @@ This guide will help you uninstall Conda from your computer. This can be useful
 
 * You have issues with packages.
 
+* You already have the Anaconda distribution installed, but need to switch to miniconda. 
+
 
 {{ windows }}
 -------------------------------------------
 
-1. Open your terminal and run the following lines, one at a time
-
-   .. code:: pwsh
-       
-       conda install anaconda-clean
-       anaconda-clean --yes
-
-   .. tip::
-      To run the above command you need to be in the ``base`` environment.
-      If it does not say ``(base)`` at the beginning of the line,
-      run the following command:
-
-      .. code:: pwsh
-         
-         conda activate base
+1. Ensure all shells, command promts are closed i.e. make sure Anaconda (or related files) aren't open anywhere.  
```

Comment:
typo, `prompts`. Also typically in writing one should leave out contractions. So spell out `are not`

Reply:
changed it